app/controllers/metadata_presenter/answers_controller.rb

```diff
@@ -67,6 +116,16 @@ def render_validation_error
       render template: page.template, status: :unprocessable_entity
     end
 
+    def incoming_answer
```

Comment:
The multi upload component is the first to treat incoming answers as additional to the existing ones for that component. It's meant treating them as a special case in a lot of places, and if we implement further components that behave this way we might consider refactoring this.